src/main/resources/payload-template

```diff
@@ -64,19 +61,13 @@
         "consensus_sequence_software_version": ${consensus sequence software version},
         "metrics": {
           "breadth_of_coverage": ${breadth of coverage value},
-          "depth_of_coverage": ${depth of coverage value},
-          "consensus_genome_length": ${consensus genome length},
-          "Ns_per_100kbp": ${Ns per 100 kbp}
+          "depth_of_coverage": ${depth of coverage value}
         },
         "reference_genome_accession": ${reference genome accession},
         "bioinformatics_protocol": ${bioinformatics protocol}
       },
-      "lineage_analysis": {
-        "lineage_name": ${lineage/clade name},
-        "lineage_analysis_software_name": ${lineage/clade analysis software name},
-        "lineage_analysis_software_version": ${lineage/clade analysis software version},
-        "variant_designation": ${variant designation},
-        "variant_evidence": ${variant evidence},
-        "variant_evidence_details": ${variant evidence details}
+      "pathogen_diagnostic_testing": {
+        "gene_name_1": ${gene name 1},
+        "diagnostic_pcr_protocol_1": ${diagnostic pcr Ct value 1}
```

Comment:
@lindaxiang what is the significance of the `_1` in this name, is this meant to be an array of objects?

Reply:
@lepsalex , when doing `Pathogen diagnostic testing`, users can choose different genes in the diagnostic RT-PCR test. Here the user are requested to report `gene name 1` and `diagnostic pcr protocol 1`.

Reply:
ok sounds good, is there anyways we could do `_one` ... I am worried about eventually parsing these fields from a central json as the `_1` would normally be parsed as an array item

Reply:
@lepsalex , yes. we can use `gene_name_one` instead of `gene_name_1` as field name in JSON payload. 
The same will do for `diagnostic_pcr_protocol_one` to replace `diagnostic_pcr_protocol_1`. 
To be clear submission TSV will keep using `gene name 1` and `diagnostic pcr Ct value 1` as field names. 
payload JSON will use `gene_name_one` and `diagnostic_pcr_protocol_one`.

Reply:
is there any way to have it be uniform?

Reply:
This needs to coordinate with DataHarmonizer's curation team. Let me confirm with Christina.

Reply:
Use `gene_name` and `diagnostic_pcr_ct_value`.